src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -104,9 +105,13 @@ public UUID printLatestGeneralLetter(CaseDetails caseDetails) {
         return bulkPrintDocuments(caseDetails.getId(), FINANCIAL_REMEDY_GENERAL_LETTER, asList(latestGeneralLetter));
     }
 
-    public List<BulkPrintDocument> approvedOrderCollection(Map<String, Object> data) {
+    public List<BulkPrintDocument> approvedOrderCollection(CaseDetails caseDetails) {
+        Map<String, Object> data = caseDetails.getData();
         List<BulkPrintDocument> bulkPrintDocuments = new ArrayList<>();
-        List<Map> documentList = ofNullable(data.get(APPROVED_ORDER_COLLECTION))
+        Object collection = CommonFunction.isConsentedInContestedCase(caseDetails)
```

Comment:
can we give this a specific type? Using Object could leave us open to a world of pain in PROD since it could return anything